query/transformation.go

```diff
@@ -42,7 +42,7 @@ func transformTimeseries(series api.Timeseries, transform transform, parameters
 // timerangeScale takes an api.Timerange and computes the "scale" or duration of one sample (in seconds).
 // It is useful for transformations that normalize on time (like derivative or integral).
 func timerangeScale(timerange api.Timerange) float64 {
-	return float64(timerange.Resolution)
+	return float64(timerange.Resolution())
```

Comment:
This was the intent of the `timerangeScale` function. Transformation functions like `derivative` treat rates as though they are per-second (rather than per-millisecond) and so are given a "scale" whose units are seconds.

The intent was that series would then take on values like "3 per second" (stored directly as `float64(3)`) instead of "0.003 per millisecond". I'm not entirely sure how this behavior will interact with the actual visualization/reporting of the data.